packages/viz/src/lib/basemap/create-map.ts

```diff
@@ -63,5 +85,7 @@ export function createMap(options: DeckGLMapOptions = DEFAULT_OPTIONS) {
     controller: true
   });
 
+  deckMap.getMapType = () => MapType.MAPBOX_GL;
```

Comment:
Adding a custom function to an instance is something that we shouldn't probably do. We need to find a property within the map that defines the kind of map and use it.